CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ project(ABACUS
     LANGUAGES CXX
 )
 
-option(ENABLE_DEEPKS "Enable DeePKS functionality" OFF)
+option(ENABLE_DEEPKS "Enable DeePKS functionality" ON)
```

Comment:
Please do not changing the default behavior of abacus.

Reply:
sure